redis.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Redis",
+    "release": "12.2-RELEASE",
+    "artifact": "https://github.com/kennedyoliveira/iocage-redis-plugin",
+    "properties": {
+        "dhcp": 1,
+        "boot": "on"
```

Comment:
This is not really required and is on by default

Reply:
ah ok, removed.